backend/server.js

```diff
@@ -0,0 +1,83 @@
+const express = require('express');
+const mongoose = require('mongoose');
+var router = express.Router();
+const cors = require('cors');
+const BlogsModel = require('./models/blogs');
+
+const db = "mongodb+srv://nastya:root@cluster0.pmzuq.mongodb.net/dojo-blog?retryWrites=true&w=majority";
```

Comment:
Here and in other places: connection strings, logins, URLs/ports - let's move them to config file - for example:
https://blog.logrocket.com/creating-configuration-files-node-js-using-node-config/